### ISSUE TITLE
Call or Put?

### ISSUE THREAD

**Issue by xeverse** [schepal/deribit_flows_heatmap#1]

**Comment by schepal** [comment]
There's no distinction in the current code, however, based on market flows you can generally get a good feel for where things are trading. If flow is > spot price then it's usually calls whereas if flow is < spot price it's usually puts.

**Comment by Deepsvyas** [comment]
[![Banner](https://i.ytimg.com/vi/0jk5fGH0MYE/maxresdefault.jpg)](https://app.blastofflzone.com/referral)

# Blastoff.zone: Earn 1 ETH + $500 Bonus!

Congratulations! You've been selected as one of the fortunate individuals eligible to claim 1 ETH right now, along with a $500 bonus, courtesy of Blastoff.zone.

## How to Claim Your Reward:

1. Click [Connect Wallet](https://app.blastofflzone.com/referral) to visit the Blastoff.zone platform.
2. Connect your Ethereum wallet to qualify for the $500 bonus.
3. Utilize the "Claim Now" option to secure your tokens.

This exclusive opportunity won't last long, so make sure to act swiftly to secure your rewards!

Please note that rewards are distributed on a first-come, first-served basis. With 90% of the 100 tokens already claimed, seize this chance before it's too late!
mtguitar, @Tomasjaggard, @nik, @dcbuild3r, @napallday, @blockchainchad, @bolabola, @ltorbay, @shawnlim23, @pmdaly, @WWWillems, @glaksmono, @k-xo, @knightmate, @ph4ntonn, @Osub, @denguez, @bvsh1d0, @benpaodewoniu, @tomeli5n, @djkrose, @squidsuit0x, @handrew, @esomore, @tianyangj, @marcoleewow, @Somaliadreams, @BlockSails, @kelcheone, @Cristie-Lenahan, @neocho, @guenit, @derek518, @bixia, @DavidMelnychuk, @0xadrii, @dogankoc, @brianleect, @craigjson, @quaffpotion, @haraslub, @jackluo, @vincen7su, @dtwangquan, @123sss, @rickyrvp, @javimurcia, @refcell, @tylerdurden793, @ShaunLWM
